collector/falcon.go

```diff
@@ -0,0 +1,17 @@
+package collector
+
+import falcon "git.n.xiaomi.com/falcon-sdk/goperfcounter"
+
+type FalconCounter struct {
+	name string
+}
+
+func (f *FalconCounter) Incr() {
+	falcon.SetCounterCount(f.name, 1)
+}
+
+func registerFalconCounter(counterName string) *FalconCounter {
```

Comment:
We also need labels `map[string]string`, it is the "tags" in terms of falcon.

```suggestion
func registerFalconCounter(counterName string, labels map[string]string) *FalconCounter {
```

Reply:
falcon doc show the multi-tag only be set in config file